safe-global/safe-wallet-web
# fix: disconnect previous wallet

## What it solves
When switching the wallet the previously connected wallet stays connected.
This can lead to many edge cases for the social signer.

## How this PR fixes it
If the user successfully switches the wallet, we disconnect the old wallet label.

## How to test it
1. Login with google
2. Click switch wallet and connect to Metamask
3. Disconnect Metamask
4. you are no longer connected to google automatically

## Checklist
